source/common/common/perf_annotation.h

```diff
@@ -42,6 +42,31 @@
     perf.record(category, description);                                                            \
   } while (false)
 
+/**
+ * Initiates a duration measurement which can be finished in a different scope.
+ * Please note that this kind of measurements makes sense only in a strictly
+ * sequential flow, i.e. when a new stream isn't created until the current one
+ * is destroyed.
```

Comment:
This in reference only to a particular combination of category and description, right?